lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -231,18 +262,41 @@ def handle(self, *args, **options):
         intervals = int(0.10 * total)
         if intervals > 100 and verbose:
             intervals = 101
+        if intervals < 1:
+            intervals = 1
```

Comment:
Did not have validation when intervals is smaller than 1 generating a ZeroDivisionError

Reply:
if you make the `intervals` definition: 
```
intervals = max(int(0.10 * total), 1)
```
you don't need this if check :)